test/unit/app/services/bulk_republisher_test.rb

```diff
@@ -0,0 +1,27 @@
+require "test_helper"
+
+class BulkRepublisherTest < ActiveSupport::TestCase
+  extend Minitest::Spec::DSL
+
+  describe "#republish_all_organisation_about_us_pages" do
+    test "queues Organisation 'About Us' pages for republishing" do
+      about_us_page_1 = create(:about_corporate_information_page)
+      about_us_page_2 = create(:about_corporate_information_page)
+      sequence = sequence("queue")
+
+      PublishingApiDocumentRepublishingWorker.expects(:perform_async_in_queue).with(
+        "bulk_republishing",
+        about_us_page_1.id,
+        true,
+      ).in_sequence(sequence)
+
+      PublishingApiDocumentRepublishingWorker.expects(:perform_async_in_queue).with(
+        "bulk_republishing",
+        about_us_page_2.id,
+        true,
+      ).in_sequence(sequence)
+
+      BulkRepublisher.new.republish_all_organisation_about_us_pages
+    end
+  end
```

Comment:
I think the issue must be that some other tests are creating about us pages that are not cleaned up. Running this test in isolation is fine, but when running the full suite, another ID is expected to be passed to the worker

Reply:
So I think the actual issue was that it need to be the `document_id` not the `id`. The lack of cleanup curiously seems not to matter